casepro/statistics/tests.py

```diff
@@ -337,6 +337,50 @@ def test_user_export(self):
         self.assertExcelRow(cases_closed_sheet, 1, [d1, 0, 0, 0, 0], tz=tz)
         self.assertExcelRow(cases_closed_sheet, 15, [d2, 0, 0, 0, 0], tz=tz)
 
+    @override_settings(CELERY_ALWAYS_EAGER=True, CELERY_EAGER_PROPAGATES_EXCEPTIONS=True, BROKER_BACKEND='memory')
+    def test_manager_user_export(self):
+        url = reverse('statistics.dailycountexport_create')
+
+        tz = pytz.timezone("Africa/Kampala")
+        d1 = date(2016, 1, 1)
+        d2 = date(2016, 1, 15)
+
+        # Jan 1st, cases opened by User1 who's part of Nyaruka
+        with patch.object(timezone, 'now', return_value=self.anytime_on_day(d1, tz)):
+            [msg] = self.new_messages(d1, 1)
+            Case.get_or_open(self.nyaruka, self.user4, msg, 'summary', self.moh)
+
+        # Jan 15th
+        with patch.object(timezone, 'now', return_value=self.anytime_on_day(d2, tz)):
+            [msg] = self.new_messages(d2, 1)
+            Case.get_or_open(self.nyaruka, self.user4, msg, 'summary', self.moh)
+
+        self.new_outgoing(self.user1, d1, 1)  # Jan 1st
+        self.new_outgoing(self.user3, d2, 1)  # Jan 15th
+
+        # logging in as user1, who's a manager for UNICEF not Nyaruka
+        self.login(self.user1)
+
+        response = self.url_post_json('unicef', url, {'type': 'U', 'after': "2016-01-01", 'before': "2016-01-31"})
```

Comment:
This is returning with a redirect for login but I've not yet been able to figure out why.